packages/entity-browser/src/modules/entityBrowser/sagas.js

```diff
@@ -104,7 +104,7 @@ const fetchRequest = (resource, params) => {
   return fetch(`${__BACKEND_URL__}/nice2/rest/${resource}${paramString}`, options)
 }
 
-function fetchRecords(entityName, page, orderBy, limit, searchTerm) {
+export function fetchRecords(entityName, page, orderBy, limit, searchTerm) {
```

Comment:
i would move this fetch methods into a own file to seperate them from the sagas a bit more.